widget/progressbarinfinite_test.go

```diff
@@ -18,12 +18,12 @@ func TestProgressBarInfinite_Destroy(t *testing.T) {
 	bar := NewProgressBarInfinite()
 	_, found := renderers.Load(bar)
 	assert.True(t, found)
-	rend := Renderer(bar).(*infProgressRenderer)
```

Comment:
changed this around, because `Renderer(bar)` has a side effect of creating a fresh new renderer if bar has an invalid renderer.

Also, testing that Running() now works as expected on the object ... which I think might be better than having test cases that look at private state in the object instead.  What do you think ?